Gemfile

```diff
@@ -108,7 +108,8 @@ gem 'sqlite3'#,  '1.3.10'
 
 # Use Unicorn as the web server
 # FIXME: Switch to Puma for Rails 5
```

Comment:
I think this comment can be removed now

Reply:
This commented-out part of the Gemfile can also be removed:

https://github.com/dradis/dradis-ce/blob/874a6492a1294fdd9e6439c9f35209aa0fa887f5/Gemfile#L11-L13